{"name": "User with wrong data types fails validation", "status": "passed", "description": "Test that wrong data types fail validation.", "attachments": [{"name": "Schema Validation Failure", "source": "7c344e27-842b-4091-97ba-5a2467757acf-attachment.json", "type": "application/json"}, {"name": "Schema Validation Failure", "source": "0d8daac2-7073-43d6-9c3e-8d325111c5b0-attachment.json", "type": "application/json"}], "start": 1756578930955, "stop": 1756578930960, "uuid": "9217943f-bc65-4636-97fa-fab416767883", "historyId": "71818746d71d76d5be64a23650196b19", "testCaseId": "71818746d71d76d5be64a23650196b19", "fullName": "tests.test_schema_validation.TestSchemaValidation#test_wrong_data_types_fail_validation", "labels": [{"name": "story", "value": "User Schema"}, {"name": "feature", "value": "Schema Validation"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_schema_validation"}, {"name": "subSuite", "value": "TestSchemaValidation"}, {"name": "host", "value": "Dons-MacBook-Air.local"}, {"name": "thread", "value": "85337-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_schema_validation"}], "titlePath": ["tests", "test_schema_validation.py", "TestSchemaValidation"]}